chart/env/prod.yaml

```diff
@@ -377,7 +377,7 @@ search:
   uvicornNumWorkers: "2"
   nodeSelector:
     role-datasets-server-search: "true"
-  replicas: 4
+  replicas: 3
```

Comment:
nit: as 250Gi means two pods per node, I imagine we can have an even number of replicas: 2 (1 node) or 4 (2 nodes)

Reply:
well nevermind, OK to reduce to 3, let's see.